packages/accounts/vite.config.js

```diff
@@ -38,6 +38,11 @@ export default ({ mode }) => {
     build: {
       target: 'esnext',
       assetsDir: 'static/sso/assets',
+      sourcemap: true,
+    },
+
+    define: {
+      'process.env.VERSION': `"${require('./package.json').version}"`,
```

Comment:
Should we use git commit hash as the release VERSION? Because it's not a releasable package and we may forget to manually bump up the version in `package.json`.

Reply:
I will change version automatically using release.sh in CI.